### ISSUE TITLE
Case insensitive sorting in the browser view

### ISSUE THREAD

**Issue by v1kn** [cmus/cmus#593]
Is it possible to configure `cmus` in a way to use case insensitive sorting in browser view (5)? 

I cannot find any information on this in manpages.

**Comment by mani1996** [comment]
Hi
I believe this feature can be added with a case insensitive version of **entry_cmp** function in **browser.c**

I can try adding this feature. I haven't contributed to cmus yet and would like to know how to get started :)

**Comment by mahkoh** [comment]
I think cmus should probably use (or provide an option to use) the configured locale when sorting these entries.

@flyingmutant Do you know of a reason for it not to do this?

**Comment by flyingmutant** [comment]
Legacy stuff, I guess? This is kind of a niche thing; sorting properly by default is much more important.

**Comment by v1kn** [comment]
> sorting properly by default is much more important

I agree sorting is vital. And case sensitivity is probably a matter of opinion or habit. To me personally case sensitive sorting is counter-productive, and can be frustrating, when I browse through a list of hundreds entries, where half start with lower case and half with upper case.

A boolean "case sensitive sorting true/false" would be best imo. Ranger file manager has this approach in its `rc.conf`
